package.json

```diff
@@ -67,6 +68,7 @@
 		"husky": "^7.0.4",
 		"jest": "^27.5.1",
 		"lint-staged": "^12.3.3",
+		"lodash.debounce": "^4.0.8",
```

Comment:
Done.

Reply:
Thanks! (Interesting that that doesn’t result in any additions or removals in `package-lock.json` – the full lodash was already in there, but also lodash.debounce doesn’t get removed because something else still pulls it in…)